.gitignore

```diff
@@ -1,10 +1,3 @@
-###> symfony/framework-bundle ###
```

Comment:
Not sure if this should be deleted. Also it should be made in another issue or ticket. A good practice is make atomic changes over a subject or issue, then make a PR only for that, then checkout master and make another branch with other different changes or issue with other PR. Good practice to read: https://seesparkbox.com/foundry/atomic_commits_with_git
Probably this time is ok go for merge, but in future we need to organize smart for avoid conflicts ;)

Reply:
Yes, this a bad practice from my part that I do a lot, feel free to say me again if this happens another time because I need to correct that.

This link also may be a "must read" in our manual for new git contributors.